zanata-war/src/main/java/org/zanata/action/VersionGroupJoinAction.java

```diff
@@ -149,14 +152,34 @@ public String send() {
         }
     }
 
-    @AllArgsConstructor
-    public final class SelectableProject {
+    public void resetData() {
+        projectSlug = "";
+        projectVersions.clear();
+        setQuery("");
+    }
 
-        @Getter
-        private HProjectIteration projectIteration;
+    @Override
+    public List<HProject> suggest() {
+        projectVersions.clear();
```

Comment:
It looks like this method is supposed to just return a list of projects, but it is also clearing the versions. I think some code needs to be restructured so that this is not confusing, or maybe add a comment saying why it needs to clear the versions.